stable/victoria-metrics/values.yaml

```
@@ -0,0 +1,171 @@
+# Default values for victoria-metrics.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+clusterDomainSuffix: "cluster.local"
+
+vminsert:
+  replicaCount: 1
+
+  terminationGracePeriodSeconds: 60
+
+  extraArgs: {}
+
+  image:
+    repository: victoriametrics/vminsert
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  imagePullSecrets: []
+
+  service:
+    type: ClusterIP
+    port: 80
+
+  serviceMonitor:
+    enabled: false
+    namespace:
```

Some kind of weird to default to not set value
```suggestion
    # namespace: monitoring
```

---

Fixed.